src/services/wallets/tweb3.ts

```diff
@@ -0,0 +1,82 @@
+import Web3 from "web3";
+import Torus from "@toruslabs/torus-embed";
+import { provider } from "web3-core";
+
+export type BuildEnvironment =
+  | "production"
+  | "development"
+  | "binance"
+  | "testing"
+  | "lrc"
+  | "beta"
+  | undefined;
+
+// interface TorusCtorArgs in torus-embeded types
+const torusSettings = {
+  //buttonPosition?: 'top-left' | 'top-right' | 'bottom-right' | 'bottom-left'
+  //modalZIndex?: number
+  //apiKey?: string
+};
+
+// interface WhiteLabelParams in torus-embeded types
+// example data structure here: https://github.com/torusresearch/torus-embed/blob/master/examples/vue-app/src/data.js
+const popupStyling = {
+  theme: {
+    isDark: false,
+    colors: {
+      torusBrand1: "#000000",
+      torusGray2: "#FBF7F3",
+    },
+  },
+  logoDark: "https://www.projectliberty.io/assets/images/brand-icon.png",
+  logoLight: "https://www.projectliberty.io/assets/images/brand-icon.png",
+  //topupHide?: boolean
+  //featuredBillboardHide?: boolean
+  //disclaimerHide?: boolean
+  //tncLink?: LocaleLinks<string>
+  //privacyPolicy?: LocaleLinks<string>
+  //contactLink?: LocaleLinks<string>
+  //customTranslations?: LocaleLinks<any>
+};
+
+// interface TorusParams in trous-embeded types
+const initSettings = (buildEnv: BuildEnvironment) => {
+  return {
+    buildEnv: buildEnv || "production",
```

Comment:
maybe we can have a config file?

Reply:
yeah that should definitely be done at some point. I'm going to add a story about adding a config in.